### PR TITLE
Fix couple of broken aliases

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
It annoys me every time I want to check if a QID is valid.

# What has changed
Fixed a couple of broken aliases.

# How to test?
Well... it's been broken this long.

# Links
Trello: https://trello.com/c/V1CoClOf

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Fix toolbox aliases](https://trello.com/c/V1CoClOf/1868-fix-toolbox-aliases)

File: aliases.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@ alias findbad='curl -s http://$EXCEPTIONMANAGER_HOST:$EXCEPTIONMANAGER_PORT/badm
 alias viewskipped='curl -s http://$EXCEPTIONMANAGER_HOST:$EXCEPTIONMANAGER_PORT/skippedmessages | jq'
 alias resetexceptionmanager='curl -s http://$EXCEPTIONMANAGER_HOST:$EXCEPTIONMANAGER_PORT/reset'
 alias msgwizard='python -m toolbox.message_tools.bad_message_wizard'
-alias qidcheck='python qid_checksum_validator.py --modulus $QID_MODULUS --factor $QID_FACTOR'
-alias qidlink='python questionnaire_linking.py'
+alias qidcheck='python python -m toolbox.qid_checksum_validator --modulus $QID_MODULUS --factor $QID_FACTOR'
+alias qidlink='python python -m toolbox.questionnaire_linking'
```
Comment (ryangrundy7) [inline aliases.sh]: You've accidentally put python twice

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
See comment

File: aliases.sh (hunk 1 above)
Comment (NickGrantX) [reply aliases.sh]: ach

well spotted thx

**Review by ryangrundy7 — APPROVED** [review]
Looks good! 👍

**Review by LeoKHoward — APPROVED** [review]

**Pull request merged**